lib/screens/explore_screens/unplant_seeds/interactor/usecases/get_planted_balance_use_case.dart

```diff
@@ -5,9 +5,11 @@ import 'package:seeds/datasource/remote/api/planted_repository.dart';
 class GetPlantedBalanceUseCase {
   final PlantedRepository _plantedRepository = PlantedRepository();
 
-  Future<Result> run() {
```

Comment:
Why is this packed into the same use case - why isn't there a separate claims use case?

So these 2 things can be loaded separately and update asynchronously? 

Getting the claim balances is potentially going to take some time.

Reply:
We usually do this when we load screen data, right now I have full loading screen, no point for now to have them separately.